packages/composites/proof-of-reserves/src/endpoint/reserves.ts

```diff
@@ -38,13 +39,14 @@ const inputParameters: InputParameters = {
 
 export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
   const validator = new Validator(input, inputParameters, paramOptions)
-
   const jobRunID = validator.validated.jobRunID
   const protocol = validator.validated.data.protocol.toUpperCase()
   const indexer: Indexer = validator.validated.data.indexer.toUpperCase()
   const confirmations = validator.validated.data.confirmations
 
   const protocolOutput = await runProtocolAdapter(jobRunID, context, protocol, input.data, config)
+  protocolOutput.result = validateAddresses(indexer, protocolOutput.result)
+  protocolOutput.data.result = protocolOutput.result
```

Comment:
Can we store the validated output as a new variable instead of overriding the existing one?